### DISCUSSION TITLE
Unfortunately, we weren't able to approve your educational discount request.

### DISCUSSION THREAD

**Discussion by AliAmmarNaseem** [community/community#52169] [GitHub Education]
I have been trying to apply for the student developer pack for many days now, but unfortunately, I have been rejected 20 times after uploading an image. I have tried uploading different documents like my student ID card, LMS screenshot, fee challan and semester schedule but still not getting approval.

I am attaching a real image issued by my institute which is not blurry and still facing problems with approval. 

**When I upload from gallery, they say :**
"The item you uploaded is insufficient to demonstrate your current academic status.
Please see suggestions.
Suggested proof to expedite your approval:
Please consider using your device camera to submit academic affiliation documents. Uploaded images are more easily manipulated and are therefore less trustworthy."
 
**When I use device camera and capture image then get following error:**
"Unfortunately, we weren't able to approve your educational discount request for {my username}

What happened?
Common verification problems and suggested solutions are documented here: https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/why-wasnt-my-application-to-global-campus-for-students-approved."

**Things I have tried:**

- Image of all documents are legible and texts are clear and easily readable. 
- Image taken is of larger resolution.
- Tried different type of documents like Student ID Card, Result card, Fee challan, Semester time table
- All Documents are in English.
- Name, username of my GitHub profile as same as my University ID card and other documents.
- My profile is set to public.

What should I do now? Please help me!

Thank you for your time and consideration.

![Screenshot 2023-04-06 175701](https://user-images.githubusercontent.com/114722213/230385500-fd8579d7-48b3-440c-a136-36635f6406bb.png)
![Screenshot 2023-04-06 175720](https://user-images.githubusercontent.com/114722213/230385510-e37162ed-7988-450c-91c3-d3546d340999.png)

**Comment by IBRAHIMtaiyeroheemat** [comment]
I am facing same problem. Please help

**Reply by AliAmmarNaseem** [reply]
I wish there was real human doing this verification, This is really annoying!!!

**Reply by ptqnhan** [reply]
I'm having the same issue. Please help!

**Comment by sarimshoukat** [comment]
I have been getting the same problem. People on different forums are also saying this is their way of avoiding the student discounts.
